src/styles/nnfx-dark.css

```diff
@@ -0,0 +1,106 @@
+/**
+ * nnfx dark - a theme inspired by Netscape Navigator/Firefox
+ *
+ * @version 1.0.0
+ * @author (c) 2020 Jim Mason <jmason@ibinx.com>
+ * @license https://creativecommons.org/licenses/by-sa/4.0  CC BY-SA 4.0
+ */
+
+.hljs {
+  display: block;
+  overflow-x: auto;
+  padding: 0.5em;
+  background: #333;
+  color: #fff;
+}
+
+.xml .hljs-meta {
```

Comment:
Curious, only xml meta?

Reply:
Yes, that one is intended to catch HTML DTD/DOCTYPEs and xml declarations only.

Reply:
Alright, your theme. :)